.github/ISSUE_TEMPLATE/8.6-request.yaml

```diff
@@ -0,0 +1,45 @@
+name: 8.6 Documentation request
+description: Request a documentation change or enhancement for Elastic Stack version 8.6.
+title: "[REQUEST]: "
+labels: ["8.6-request", "Team:Docs"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this request! This form will create an issue that the obs-docs team will triage and prioritize. Filling out this form does not guarantee that your issue will be prioritized for the selected iteration.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe what needs to be documented.
+      placeholder: I need...
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Resources
+      description: Please link to as many related resources as possible. Think about issues, PRs, demos, Slack threads, etc.
+      placeholder: How can the docs team learn more about this feature?
+    validations:
+      required: true
+  - type: checkboxes
+    id: collaboration
+    attributes:
+      label: Collaboration
+      description: Choose the expected collaboration model
+      options:
+        - label: The **documentation** team will proide the initial content.
```

Comment:
```suggestion
        - label: The **documentation** team will investigate the issue and create the initial content.
```

Reply:
I was going to just suggest a simple typo fix but I wonder if it's worth rephrasing to emphasis more precisely what we would do?